navigation/ca_move_base/maps/empty.yaml

```diff
@@ -0,0 +1,6 @@
+free_thresh: 0.196
+image: playground.pgm
```

Comment:
[This parameter](http://wiki.ros.org/map_server#YAML_format) should point to a `pgm` file that corresponds to an empty environment.

I recommend you to compare you approach with this one ([empty_world](https://github.com/utexas-bwi/segbot_rocon/tree/bc96568f23b917d80744c6730d3adb76b31b2f55/segbot_concert_services/maps/empty_world)) and also include the missing `pgm`.